hoot-core/src/main/cpp/hoot/core/algorithms/splitter/IntersectionSplitter.cpp

```diff
@@ -166,7 +165,7 @@ void IntersectionSplitter::splitIntersections()
 
   if (Log::getInstance().isInfoEnabled() && todoLogged)
   {
-    cout << endl;
+    LOG_INFO("  Intersection splitter todo: 0       ");
```

Comment:
For the `cout << endl` replacements I chose to write out the last `PROGRESS_INFO` call but with `LOG_INFO` which writes a carriage-return and line feed instead of just a carriage-return.